models/clipper/ethereum/clipper_ethereum_schema.yml

```diff
@@ -0,0 +1,133 @@
+version: 2
+
+models:
+  - name: clipper_v1-3_ethereum
```

Comment:
i believe this would apply to model name (aka file name), not schema applied, right?

Reply:
if so, would need split out for each version